R/bodyFooter.R

```diff
@@ -19,27 +19,64 @@
 
 
 # Module Server Function
-.bodyFooter <- function(input, output, session, logdata) {
-    output$dt_userlog <- shiny::renderTable({
+.bodyFooter <- function(..., logdata) {
+    call <- match.call()
+    params <- list(...)
+    param_index <- 1
+    params_length <- length(params)
+    old_style_call <- call[[1]] == "module" || "periscope" %in% as.character(call[[1]])
+    
+    # get session parameters
```

Comment:
remove comment

Reply:
- done, for the comment part.
- for extracting the logic into a separate function, that is what I want to do first time. But for some reason, seasion parameters got corrupted if I used them in a do.call if I build the functions arguments dynamically or passed them as a list to use them. So I had to let each function build its own arguments using the passed session parameter as they are with no change or indirect calls. Thank you